Lib/fontTools/colorLib/builder.py

```diff
@@ -487,115 +610,50 @@ def buildPaintRadialGradient(
     return self
 
 
-def buildPaintGlyph(glyph: str, paint: _PaintInput) -> ot.Paint:
+def buildPaintGlyph(layerCollector: LayerCollector, glyph: str, paint: _PaintInput) -> ot.Paint:
```

Comment:
I wonder if we can make this layerCollector an optional keyword argument that defaults to None, and when None these buildPaintSomething would just make a new LayerCollector and pass it down. E.g. if one only want to build a specific paint table that doesn't need to refer to the global LayerV1List.

Reply:
Would it be too disruptive to propose we have a class akin to ColrBuilder and hang the various buildSomething method on it? - akin to FontBuilder. It's a model that makes a lot of sense for a series of related and stateful calls, which is now true here.

Reply:
> have a class akin to ColrBuilder

yeah, I like that.

Reply:
Updated